### ISSUE TITLE
How to remove featured list restriction of 7 days

### ISSUE THREAD

**Issue by GreenArrowlol** [tomo54321/McServer-List#8]
I noticed that it doesn't let people pay for featured slot unless their server is of 7 days old but that might not be good in some situations.

So please let me know on how to remove this restriction.
![image](https://github.com/user-attachments/assets/cdddf1b3-c1ac-4d39-9635-4ffe39f8af4e)

**Comment by tomo54321** [comment]
Hi,
To remove this you are going to need to go and remove the checks and references to this within the views and controllers. 
The ones I have been able to find quickly are:
https://github.com/tomo54321/McServer-List/blob/master/app/Http/Controllers/FeatureController.php#L36
https://github.com/tomo54321/McServer-List/blob/master/resources/views/featured/show.blade.php#L27

**Comment by GreenArrowlol** [comment]
> Hi, To remove this you are going to need to go and remove the checks and references to this within the views and controllers. The ones I have been able to find quickly are: https://github.com/tomo54321/McServer-List/blob/master/app/Http/Controllers/FeatureController.php#L36 https://github.com/tomo54321/McServer-List/blob/master/resources/views/featured/show.blade.php#L27

What about just reducing the minimum requirement to 1 day?

How to do that?

**Comment by tomo54321** [comment]
This will still involve editing the files sent above to lower the requirement from 1 week (7 days) to 1 day.

**Comment by GreenArrowlol** [comment]
> This will still involve editing the files sent above to lower the requirement from 1 week (7 days) to 1 day.

I see, can you point out the exact value we need to enter there? I see that it says addWeeks(-1)) but not sure about how to make it to a day.

**Comment by tomo54321** [comment]
Yes that the line you need to edit, you are able to manipulate the date however, you choose from there. It is using a Carbon date object, you need to change it to be addDays, here's the section in their docs you'll need to take a look at to change it to a day, week, month or year. 

https://carbon.nesbot.com/docs/#api-addsub

**Comment by GreenArrowlol** [comment]
> Yes that the line you need to edit, you are able to manipulate the date however, you choose from there. It is using a Carbon date object, you need to change it to be addDays, here's the section in their docs you'll need to take a look at to change it to a day, week, month or year.
> 
> https://carbon.nesbot.com/docs/#api-addsub

Thanks.

Can I also know how to change this to 1?
![image](https://github.com/user-attachments/assets/868ea249-5db5-4874-af01-76ce6df03776)

I tried looking but don't see what to edit.

**Comment by GreenArrowlol** [comment]
> > Yes that the line you need to edit, you are able to manipulate the date however, you choose from there. It is using a Carbon date object, you need to change it to be addDays, here's the section in their docs you'll need to take a look at to change it to a day, week, month or year.
> > https://carbon.nesbot.com/docs/#api-addsub
> 
> Thanks.
> 
> Can I also know how to change this to 1? ![image](https://private-user-images.githubusercontent.com/76021102/357112064-868ea249-5db5-4874-af01-76ce6df03776.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjM0Nzc0MzMsIm5iZiI6MTcyMzQ3NzEzMywicGF0aCI6Ii83NjAyMTEwMi8zNTcxMTIwNjQtODY4ZWEyNDktNWRiNS00ODc0LWFmMDEtNzZjZTZkZjAzNzc2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MTIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODEyVDE1Mzg1M1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWM0ZmQwZTQwOTkxMzczODEwZGI1NTQ4MDUzNWY0MWRiOWIxZjE1N2U1NTlmNWFiZGU0YjBhZWE0OGIxM2JlOTUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.pPu5P4NdUq6EbjHMYQPu8mvzjQ9cysqCp7q1sOaQrPE)
> 
> I tried looking but don't see what to edit.

Found it
`    'minsponsordays' => 1, // Minimum days you can sponsor your server for`

![image](https://github.com/user-attachments/assets/f8b9d882-581b-488c-b94e-979bfd9ca613)
